docs/docs/auth/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Authentication
+sidebar_position: 10
+---
+
+TODO general information about how we handle authentication
+
+## Available Decorators
+
+### @PublicApi
+
+`@PublicApi()` can be used to expose a handler (one query, mutation or route) or a whole class (resolver or controller) publicly.
+
+:::caution
+
+Use with caution at class level because later added handlers are automatically public.
+
+:::
+
+### @DisableGlobalAuthGuard
+
+`@DisableGlobalAuthGuard()` can disable the global auth guard (`CometAuthGuard`). This can be used if a different authentication method is desired (e.g., basic authentication). Assign a class a custom guard with `@UseGuards`. The custom guard can leverage `@PublicApi` as well to expose handlers publicly.
```

Comment:
Should we add an example for this? And be more insistent that a custom guard should be implemented instead?

Reply:
```suggestion
`@DisableGlobalAuthGuard()` disables the global auth guard (`CometAuthGuard`). This may be used if a different authentication method is desired (e.g., basic authentication) for a specific handler or class. It should be used in combination with a custom guard. The custom guard may leverage `@PublicApi` as well to expose handlers publicly.
```

Reply:
Example:

```ts
@DisableGlobalGuard()
@UseGuards(MyCustomGuard)
async handlerThatUsesACustomGuard(): {
  ...
}
```
```